packages/dbml-cli/src/cli/index.js

```diff
@@ -39,7 +40,27 @@ function sql2dbml (args) {
   importHandler(program);
 }
 
+function db2dbml (args) {
+  program.version(projectInfo.version);
+
+  program
+    .usage('[options]')
+    .option('--postgres', 'connect to a postgresql database')
+    .option('--database <name>', 'database name')
+    .option('--host <host>', 'database host. default: localhost')
+    .option('--port <port>', 'database port. default: 5432')
+    .option('--user <user>', 'database user')
+    .option('--password <password>', 'database password')
+    .option('--connection-string <connectionString>', 'database connection string')
+    .option('-o, --out-file <pathspec>', 'compile all input files into a single files');
```

Comment:
We can remove the need to use `--connection-string` before the connection string by doing this:

```
program
    .usage('<connection-string> [options]')
    .arguments('database connection string')
    .option(...)
    ....
```

And then in `connectionHandler`, you can read the connection-string through the programs.args (it is an array)